common/data/base/houseBase.ts

```diff
@@ -1,68 +1,17 @@
-import qs from 'qs';
 import { HistogramDataWithoutValues } from 'components/UI/RangeHistogram';
-import { getObjectsTotalCount, parallelRequests, STRAPI_BASE_URL } from '../dataHelpers';
-import { FilterOperator } from '../filterOperator';
-import { getById } from './getById';
 
 export const houseBase = {
-    async getObject(id: string): Promise<HouseObject> {
-        return getById.getObject(id, '/house');
-    },
-    async getObjectsPolygonsByRange(
-        from: number,
-        to: number,
-        filterName: string,
-    ): Promise<HouseClient[]> {
-        const url = `${STRAPI_BASE_URL}/house`;
-
-        const query = qs.stringify({
-            filters: {
-                [filterName]: {
-                    [FilterOperator['>=']]: from,
-                    [FilterOperator['<=']]: to,
-                },
-            },
-            populate: 'borders',
-            fields: filterName,
-        });
-
-        const result = await parallelRequests(
-            `${url}?${query}`,
-            (x: HouseObject): HouseClient => ({
-                borders: x.attributes.borders?.coordinates,
-                year: x.attributes.Year,
-                floors: x.attributes.Floors,
-                wearAndTear: x.attributes.WearAndTear,
-                id: x.id,
-            }),
-        );
-
-        return result;
-    },
-    async getFilterValues(histogramData: HistogramDataWithoutValues, filterName: string) {
-        const requests: Promise<number>[] = [];
-
-        histogramData.forEach(({ from, to }, idx) => {
-            const toFilter = idx === histogramData.length - 1 ? FilterOperator['<='] : FilterOperator['<'];
-
-            const url = `${STRAPI_BASE_URL}/house`;
-
-            const query = qs.stringify({
-                filters: {
-                    [filterName]: {
-                        [FilterOperator['>=']]: from,
-                        [toFilter]: to,
-                    },
-                },
-                fields: 'Address',
-            });
-
-            requests.push(getObjectsTotalCount(`${url}?${query}`));
-        });
-
-        const result = await Promise.all(requests);
-
-        return result;
+    async getFilterValues(_: HistogramDataWithoutValues, filterName: string) {
+        switch (filterName) {
+            case 'Year':
+                return Promise.resolve([0, 8, 67, 195, 982, 4790, 935, 811]);
```

Comment:
tbd: move values generation to build step

Reply:
Outdated thing. right now I get them by sql query:
https://github.com/ekaterinburgdev/map/blob/10897744bc0b9dd2424bd6cde2e5744b77bab95a/scripts/getHousesMeta.js